libmproxy/proxy/config.py

```diff
@@ -133,10 +133,12 @@ def process_proxy_options(parser, options):
 
     if options.clientcerts:
         options.clientcerts = os.path.expanduser(options.clientcerts)
-        if not os.path.exists(options.clientcerts) or not os.path.isdir(options.clientcerts):
+        if not (os.path.exists(options.clientcerts) or
```

Comment:
This condition doesn't make much sense, `isdir/isfile` always imply ´.exists`. I think just checking for `.exists()` is good enough.